dashboard/app/dsl/level_group_dsl.rb

```diff
@@ -21,6 +21,14 @@ def level(name)
     @hash[:levels] << name
   end
 
+  def submittable(text)
```

Comment:
The shortcuts seem to add it to the object, not the hash, so then we'd need to change the implementation of parse_output.  Some advantages, but I dislike having functionality spread between a base class and its descendent (composition vs. inheritance debate of course) and with relatively few comments on this system I find the split a bit tricky to parse (hah).